src/python/skoleklassen/skoleklassen.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Skoleklassen
```

Comment:
Du kan gjerne lage en egen branch, og lage to pull request når du har to oppgaver (så fremst de ikke er avhengige av hverandre).